src/scripts.js

```diff
@@ -65,49 +65,44 @@ function instantiateClasses(userData, ingredData, recipeData) {
   currentUser = new User(userData[0]);
   displayRecipe(recipeRepo.repo[0]);
   viewAllRecipes(recipeRepo.repo);
-};
+}
 
 function fetchAllData() {
-  Promise.all([fetchData('users'), fetchData('ingredients'), fetchData('recipes')])
-    .then(data => {
-      userData = data[0].usersData
-      ingredData = data[1].ingredientsData
-      recipeData = data[2].recipeData
-      instantiateClasses(userData, ingredData, recipeData);
+  Promise.all([
+    fetchData("users"),
+    fetchData("ingredients"),
+    fetchData("recipes"),
+  ]).then((data) => {
+    userData = data[0].usersData;
+    ingredData = data[1].ingredientsData;
+    recipeData = data[2].recipeData;
+    instantiateClasses(userData, ingredData, recipeData);
   });
-};
+}
 fetchAllData();
 
-
-
-
-
 function favoriteRecipe() {
   if (!currentUser.favoriteRecipes.includes(currentRecipe)) {
     currentUser.addToFavoriteRecipes(currentRecipe);
     favoriteBtnStar.src = "./images/star-icon-red.png";
-  };
-};
+  }
+}
 
 function viewFavoriteRecipes() {
-  viewAllRecipes(currentUser.favoriteRecipes)
-  searchFavoritesBtn.classList.toggle('hidden')
-  searchButton.classList.toggle('hidden')
-  filterAllBtn.classList.remove('hidden')
-  favoriteFilterBtn.classList.add('hidden')
+  viewAllRecipes(currentUser.favoriteRecipes);
+  seeFavoritesView();
 }
 
 function allRecipes() {
-  viewAllRecipes(recipeRepo.repo)
-  filterAllBtn.classList.toggle('hidden')
-  favoriteFilterBtn.classList.toggle('hidden')
+  viewAllRecipes(recipeRepo.repo);
+  seeAllView();
 }
 
 //searchBox.value === recipeRepo.forEach((recipe) => recipe.name)
 
 function searchRecipe() {
```

Comment:
Because of the way this function is built you're only searching for title OR tag if there isn't any matched titles in the recipe list. If this is intended and you only want to search tags if there isn't a match in the title that's okay, just wanted to bring this to attention.